tests/external-legacy-data/v0_11.sql

```diff
@@ -0,0 +1,138 @@
+/*!40000 ALTER TABLE `~external` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `~log`
+--
+
+DROP TABLE IF EXISTS `~log`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `~log` (
+  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `version` varchar(12) NOT NULL COMMENT 'datajoint version',
+  `user` varchar(255) NOT NULL COMMENT 'user@host',
+  `host` varchar(255) NOT NULL DEFAULT '' COMMENT 'system hostname',
+  `event` varchar(255) NOT NULL DEFAULT '' COMMENT 'custom message',
+  PRIMARY KEY (`timestamp`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1 COMMENT='event logging table for `djtest_blob_migrate`';
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `~log`
+--
+
+LOCK TABLES `~log` WRITE;
+/*!40000 ALTER TABLE `~log` DISABLE KEYS */;
+INSERT INTO `~log` VALUES ('2019-07-31 17:54:49','0.11.1py','root@172.168.1.4','297df05ab17c','Declared `djtest_blob_migrate`.`~log`'),('2019-07-31 17:54:54','0.11.1py','root@172.168.1.4','297df05ab17c','Declared `djtest_blob_migrate`.`~external`'),('2019-07-31 17:54:55','0.11.1py','root@172.168.1.4','297df05ab17c','Declared `djtest_blob_migrate`.`b`');
+/*!40000 ALTER TABLE `~log` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `a`
+--
+
+DROP TABLE IF EXISTS `a`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `a` (
+  `id` int(11) NOT NULL,
+  `blob_external` char(51) NOT NULL COMMENT ':external:uses S3',
+  `blob_share` char(51) NOT NULL COMMENT ':external-shared:uses S3',
+  PRIMARY KEY (`id`),
+  KEY `blob_external` (`blob_external`),
+  KEY `blob_share` (`blob_share`),
+  CONSTRAINT `a_ibfk_1` FOREIGN KEY (`blob_external`) REFERENCES `~external` (`hash`),
+  CONSTRAINT `a_ibfk_2` FOREIGN KEY (`blob_share`) REFERENCES `~external` (`hash`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `a`
+--
+
+LOCK TABLES `a` WRITE;
+/*!40000 ALTER TABLE `a` DISABLE KEYS */;
+INSERT INTO `a` VALUES (0,'_3A03zPqfVhbn0rhlOJYGNivFJ4uqYuHaeQBA-V8PKA','NmWj002gtKUkt9GIBwzn6Iw3x6h7ovlX_FfELbfjwRQshared'),(1,'_Fhi2GUBB0fgxcSP2q-isgncIUTdgGK7ivHiySAU_94','FoRROa2LWM6_wx0RIQ0J-LVvgm256cqDQfJa066HoTEshared');
+/*!40000 ALTER TABLE `a` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `b`
+--
+
+DROP TABLE IF EXISTS `b`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `b` (
+  `id` int(11) NOT NULL,
+  `blob_local` char(51) NOT NULL COMMENT ':external-local:uses files',
+  `blob_share` char(51) NOT NULL COMMENT ':external-shared:uses S3',
+  PRIMARY KEY (`id`),
+  KEY `blob_local` (`blob_local`),
```

Comment:
Interesting. These keys are redundant since the foreign key declaration below triggers their implicit creation.

Reply:
Hey Dimitri, I see. I had not reviewed this script to closely as it was auto-generated by a `mysql dump ...` command. Is this required for us to merge on this feature? Trying to avoid changes on the generated file simply to streamline the process of creating/modifying this test.

Reply:
ok, no problem